src/objects.js

```diff
@@ -1,41 +1,64 @@
 const createPerson = (name, age) => {
-  // your code here
+  return {name, age
+  };
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
-};
+  if (object.hasOwnProperty(property)){
+  return true;
+  }
+  else {
+    return false;
+  }
+  };
+
 
 const isOver65 = person => {
-  // your code here
-};
+  if (person.age > 65) {
+    return true;
+  } else if(person.age <= 65) {
+    return false;
+  }
+  };
 
 const getAges = people => {
-  // your code here
-};
+  return people.map(person => person.age);
+  };
 
 const findByName = (name, people) => {
-  // your code here
-};
+  const person = people.find(someone => someone.name === name);
+  return person;
```

Comment:
You could just return the method straightaway you do not have to assign it to the person variable as you dont use it anywhere else apart from the return.
(like above)